modules/channel-slack/src/backend/client.ts

```diff
@@ -103,7 +103,7 @@ export class SlackClient {
       await this.bp.events.updateEvent(event.id, { feedback })
     })
 
-    this.router.use(`/bots/${this.botId}/callback`, this.interactive.requestListener())
```

Comment:
This is the breaking change for old RTM users. This line has been there since before I implemented the new slack API. Both the new API and the old API use this to listen for interactive events.

Reply:
got it. Let me check if we can simply drop RTM support